src/components/RadonOperator.vue

```diff
@@ -6,13 +6,13 @@
   >
     <div class="border">
       <el-tooltip
-        v-show="showDelete && !hideDelete"
+        v-if="showDelete && !hideDelete"
```

Comment:
Why should we replace this `v-show` with `v-if`?

Reply:
https://github.com/tailwindlabs/headlessui/issues/1709